core/templates/components/common-layout-directives/common-elements/sharing-links.component.html

```diff
@@ -110,11 +100,25 @@
   }
 
   i.oppia-share-icons.font-big {
-    font-size: 32px;
+    font-size: 2.01rem;
+    margin-top: 0.004375rem;
```

Comment:
Do we really need this margin-top (seems like quite a small value)?

Reply:
yeah, to align the icons togethers.